cogs/welcome/welcome.py

```diff
@@ -298,6 +286,28 @@ async def sendLogUserDescription(self, user: discord.Member):
                     user.id,
                 )
 
+    async def logServerJoin(self, joinUser: discord.Member):
+        """Logs the server join to a channel, if enabled."""
+        async with self.config.guild(joinUser.guild).all() as guildData:
+            if guildData[KEY_LOG_JOIN_ENABLED]:
+                channel = discord.utils.get(
+                    joinUser.guild.text_channels, id=guildData[KEY_LOG_JOIN_CHANNEL]
+                )
```

Comment:
We're reading only 2 values here so we can just do 2 `await`s to read them individually.

```suggestion
        guildConfig: Group = self.config.guild(joinUser.guild)
        if await guildConfig.get_attr(KEY_LOG_JOIN_ENABLED)():
            logJoinChannelId: Optional[int] = await guildConfig.get_attr(KEY_LOG_JOIN_CHANNEL)()
            if logJoinChannelId:
                channel: Optional[TextChannel] = discord.utils.get(
                    joinUser.guild.text_channels, id=logJoinChannelId
                )
```